core/src/main/scala/contextual/prefix.scala

```diff
@@ -35,8 +35,9 @@ object Prefix {
     * @return a new instance of a [[Prefix]]
     */
   def apply(interpolator: Interpolator, stringContext: StringContext):
-      Prefix[interpolator.ContextType, interpolator.type] =
+      Prefix[interpolator.type#Output, interpolator.ContextType, interpolator.type] =
```

Comment:
Can we write `interpolator.Output` instead of `interpolator.type#Output`? I think they should be identical.